docs/content/modules/core-and-feature-module-overview.md

```diff
@@ -48,6 +48,14 @@ Call this when you create your [worker]({{urlRoot}}/content/workers/workers-in-t
 Call this to add the SpatialOS components used by the player lifecycle module to your entity.<br/>
 Call this during [entity template creation]({{urlRoot}}/content/entity-templates).
 
+* `byte[] SerializeArguments(object playerCreationArguments)` - in the repository [here](https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/PlayerLifecycleHelper.cs).<br/>
+Call this to serialize arbitrary data into a byte array that can be passed into a player creation request.<br/>
+Call this when you manually request player creation.
+
+* `T DeserializeArguments<T>(byte[] serializedArguments)` - in the repository [here](https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/PlayerLifecycleHelper.cs).<br/>
+Call this to deserialize the byte array that was passed into a player creation request.<br/>
+Call this when you want to make use of the deserialized arguments in an [entity template]({{urlRoot}}/content/entity-templates).
```

Comment:
doesn't need to be in an entity template, right? 

```suggestion
Call this when you want to make use of the deserialized arguments in your code.
```

Reply:
Do you feel that these methods even belong here?

Reply:
This is the feature module overview - this feels much more detail that is required here

Reply:
seeing as we don't have a dedicated page for the player lifecycle feature module, I think it's fine. Though yes, we can also just remove that info completely

Reply:
nevermind that: we do have a dedicated page xD
in which case I completely agree with jamie. this shouldn't be here